middlewares/ip/README.md

```diff
@@ -0,0 +1,32 @@
+## IP Blacklisting/Whitelisting Middlewares
+
+This package provides the announce middlewares `ip_blacklist` and `ip_whitelist` for blacklisting or whitelisting IP addresses and networks for announces.
+
+### `ip_blacklist`
```

Comment:
All of our other YAML uses camelCase and not snake_case. YAML doesn't actually define anywhere which is the standard. If you think snake_case is better, open another PR that changes this across the board. But for now, let's put this in as camelCase.